web/src/user_profile.js

```diff
@@ -463,6 +463,7 @@ export function show_user_profile(user, default_tab_key = "profile-tab") {
     const $elem = toggler.get();
     $elem.addClass("large allow-overflow");
     $("#tab-toggle").append($elem);
+    $(".ind-tab").trigger("focus");
     if (show_user_subscribe_widget) {
```

Comment:
Let's use `setTimeout` to ensure that element is always rendered after we add it to DOM and perhaps we should use a more specific selector for setting focus rather than a class of elements? Maybe just using `first()` is fine here before `.trigger`.

Reply:
@amanagr I've added the `setTimeout` so that it will be executed as soon as the browser's main thread is free. Also instead of using `.ind-tab` as selector I am now using `.ind-tab.first`, I hope this is fine.